plugins/workspace-minimap/test/index.ts

```diff
@@ -12,22 +12,23 @@ import * as Blockly from 'blockly';
 import {toolboxCategories, createPlayground} from '@blockly/dev-tools';
 import {Minimap, PositionedMinimap} from '../src/index';
 
-/**
- * Create a workspace.
- * @param blocklyDiv The blockly container div.
- * @param options The Blockly options.
- * @returns The created workspace.
- */
-function createWorkspace(blocklyDiv: HTMLElement,
-    options: Blockly.BlocklyOptions): Blockly.WorkspaceSvg {
-  const workspace = Blockly.inject(blocklyDiv, options);
-
-  const minimap = new Minimap(workspace);
-  minimap.init();
-
-  return workspace;
-}
-
+// Creates the primary workspace and adds the minimap.
 const ws = Blockly.inject('primaryDiv', {toolbox: toolboxCategories});
```

Comment:
The prior names ('workspace' and 'minimap') are preferable - the styleguide generally discourages abbreviations for TS.